base_module_record/wizard/base_module_record_objects.py

```diff
@@ -71,12 +71,11 @@ def record_objects(self):
                 if "_auto" in dir(obj_pool):
                     if not obj_pool._auto:
                         continue
-            search_ids = obj_pool.search(search_condition)
-            for s_id in search_ids:
-                dbname = self.env.cr.dbname
-                args = (dbname, self.env.user.id, obj_name, 'copy',
-                        s_id.id, {})
-                recording_data.append(('query', args, {}, s_id.id))
+
+            [recording_data.append(("query", (self.env.cr.dbname, 
+                                         self.env.user.id, obj_name, 
+                                         "copy", s_id.id, {}), 
+                               {}, s_id.id)) for s_id in obj_pool.search(search_condition)]
```

Comment:
LIST unassigned to any member variable!  no still not right.
recording_data = []
recording_data += code..
check

Reply:
Done